shpc/main/modules/templates/docs.md

```diff
@@ -118,7 +118,7 @@ $ docker run --rm {% if settings.bindpaths %}-v {{ settings.bindpaths }}{% endif
 In the above, the `<container>` directive will reference an actual container provided
 by the module, for the version you have chosen to load. An environment file in the
 module folder will also be bound. Note that although a container
-might provide custom commands, every container exposes unique exec, settings.singularity_shell, run, and
+might provide custom commands, every container exposes unique exec, shell, run, and
```

Comment:
Can you double check the docs template is fed the settings? I don’t recall if I added them there or not (I’m away from computer going to sleep but can also check in ~20 hours after next word day).

Reply:
sorry, I am not sure I understand your request.
I gave a quick look inside the `docs/` files, and all `shell` occurrences seemed sensible.

Reply:
Here are the variables that are passed to the documentation generation (e.g., settings is there but not shell!)
```
# Currently one doc is rendered for all containers
        result = template.render(
            parsed_name=config.name,
            settings=self.settings,
            description=config.description,
            aliases=aliases,
            versions=config.tags.keys(),
            github_url=github_url,
            container_url=config.url,
            creation_date=datetime.now(),
            name=module_name,
            flatname=module_name.replace(os.sep, "-"),
        )
```

Reply:
ah, interesting, so `shell` only relates to docker/podman

Reply:
yeah sorry I wasn’t consistent! I thought switching to using the settings would make it slightly more so.

Reply:
done :)